TypeCobol.Analysis.Test/BasicCfgInstrs/IX105A.diag

```diff
@@ -12,18 +12,38 @@ Line 421[12,47] <37, Warning, General> - Warning: "end-if" is missing
 Line 423[12,46] <37, Warning, General> - Warning: "end-if" is missing
 Line 436[12,47] <37, Warning, General> - Warning: "end-if" is missing
 Line 437[12,42] <37, Warning, General> - Warning: "end-if" is missing
+Line 455[12,12] <37, Warning, General> - Warning: Statement '       GO TO WRITE-TEST-GF-01-FAIL' located at line 485, column 15 prevents this PERFORM statement to reach its exit.
```

Comment:
IBM Compiler doesn't report for this GOTO.
What I suspect based on test:
Englobing section `SECT-RC-02-001` is called and not the paragrah `WRITE-TEST-GF-01` which contains the GO TO.
The GO TO is inside the same section so the execution flow will continue until it reaches the end of the section.
After that the next instruction after the call to the section will be called.
Same rules applies for perform THRU, if the GO TO doesn't jump after the "THRU paragraph/section ", then it's ok.


Note that in this case, there is no perform to section `SECT-RC-02-001`. All section/paragraphs of the program are called in sequence.

Reply:
This is in fact a limitation of IBM compiler in this case because the incriminated `GO TO` is inside a `WRITE` condition. IBM compiler does not report premature `PERFORM` exit whereas our CFG implementation does.

Reply:
Yes, indeed. It works as expected.